tutorials/micro/micro_autotune.py

```diff
@@ -147,8 +149,8 @@
 #        build_func=tvm.micro.autotvm_build_func,
 #    )
 #    runner = tvm.autotvm.LocalRunner(number=1, repeat=1, timeout=100, module_loader=module_loader)
-
-# measure_option = tvm.autotvm.measure_option(builder=builder, runner=runner)
+#
+#    measure_option = tvm.autotvm.measure_option(builder=builder, runner=runner)
```

Comment:
one space?

Reply:
I think this is the tutorial format. Picked it up from other tutorials.